Guide/src/dev_guide/getting_started/suggested_dev_env.md

```diff
@@ -21,6 +21,24 @@ If you're using a different development environment, we nonetheless suggest
 reading through this section, so you can enable similar settings in whatever
 editor / IDE you happen to be using.
 
+```admonish tip
+Just want the recommended editor settings? Put this in `openvmm/.vscode/settings.json`:
+
+```json
+{
+    "rust-analyzer.linkedProjects": [
+        "Cargo.toml",
+    ],
+    "rust-analyzer.cargo.targetDir": true,
+    "rust-analyzer.imports.granularity.group": "item",
+    "rust-analyzer.imports.group.enable": false,
+    "[rust]": {
+        "editor.formatOnSave": true
+    },
+}
+```
+```
```

Comment:
this isn't going to render correctly. look at the GitHub preview.

please send out a hotfix